File: 백준/Bronze/2506. 점수계산/점수계산.cs
int len = int.Parse(Console.ReadLine());
int rs = 0, cnt = 0;

string[] arr = Console.ReadLine().Split(' ');

foreach (string s in arr)
{
    if (s.Equals("1")) cnt++;
    else cnt = 0;

    rs += cnt;
}

Console.WriteLine(rs);

File: 백준/Bronze/2506. 점수계산/점수계산.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int len = sc.nextInt();
        int rs = 0, cnt = 0;
        
        sc.nextLine(); 
        String[] arr = sc.nextLine().split(" ");
        
        for (String s : arr) {
            if (s.equals("1")) 
                cnt++;
            else 
                cnt = 0;
            
            rs += cnt;
        }
        
        System.out.println(rs);
        sc.close();
    }
}
